lib/matplotlib/tests/test_path.py

```diff
@@ -410,6 +410,16 @@ def test_path_intersect_path(phi):
     b = transform.transform_path(Path([(0, 1), (0, 2), (0, 5)]))
     assert a.intersects_path(b) and b.intersects_path(a)
 
+    # test added to adress issue #20328
```

Comment:
Can you say what #20328 was?  This code may outlive GitHub and this comment won't be helpful...

Reply:
Good suggestion, I rewrote the comments to be more alike the ones already present in the tests.